### DISCUSSION TITLE
ERROR SSL DH_KEY_TOO_SMALL

### DISCUSSION THREAD

**Discussion by kauaSantos123** [TadaSoftware/PyNFe#288] [Q&A]
Estou tendo esse erro 
requests.exceptions.SSLError: HTTPSConnectionPool(host='nfe.sefaz.go.gov.br', port=443): Max retries exceeded with url: /nfe/services/NFeConsultaProtocolo4?wsdl (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:992)')))

Ao consultar notas do estado de GO acredito que seja por causa do TLS mas nao estou conseguindo ajustar!

**Comment by leogregianin** [comment]
Certificado SSL da SEFAZ GO está com problemas, veja https://nfe.sefaz.go.gov.br/nfe/services/NFeConsultaProtocolo4?wsdl

![sefaz-go](https://github.com/TadaSoftware/PyNFe/assets/1684053/16b306a3-6d52-46d2-b6d9-654e3da13c23)

**Reply by kauaSantos123** [reply]
Desculpe mas não entendi, outro estado apresenta a mesma coisa e funciona !

**Comment by kauaSantos123** [comment]
Algum retorno?
